modules/pref/src/main/Pref.scala

```diff
@@ -113,6 +113,9 @@ case class Pref(
       highlight &&
       coords == Coords.OUTSIDE
 
+  def simpleBoard =
```

Comment:
I'm not sure the server is the right place to be setting this class. If it's just a calculation of css variables it can be done on page load. I'll already have to do it when sliders are adjusted now.